src/legacy/core_plugins/kibana/public/discover/_discover.scss

```diff
@@ -36,8 +36,18 @@ discover-app {
 }
 
 .dscResultCount {
+  display: flex;
+  justify-content: center;
```

Comment:
This is fine, but as an FYI, it's easier to just set it as `text-align: center` since it's just text inside. It's more likely to be fine across browsers.